fiboa_cli/datasets/be_wa.py

```diff
@@ -0,0 +1,89 @@
+from ..convert_utils import convert as convert_
+
+URI = "http://use_your_own_copy/"
```

Comment:
Let me think about a solution that is a bit better than this approach. Maybe we can let users input the path directly when an empty URL is provided through the template. The -c option will change in #48 anyway.